commercetools-internal-processors/src/test/java/io/sphere/sdk/annotations/processors/generators/AbstractMultipleGeneratorTest.java

```diff
@@ -0,0 +1,56 @@
+package io.sphere.sdk.annotations.processors.generators;
+
+import com.google.testing.compile.CompilationRule;
+import com.squareup.javapoet.JavaFile;
+import com.squareup.javapoet.TypeSpec;
+import org.apache.commons.io.Charsets;
+import org.apache.commons.io.IOUtils;
+import org.junit.Rule;
+
+import javax.lang.model.element.TypeElement;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class AbstractMultipleGeneratorTest {
+
+    @Rule
+    public CompilationRule compilationRule = new CompilationRule();
+
+    protected AbstractMultipleGenerator generator;
+
+    protected List<String> expectedContent(final Class<?> clazz) throws Exception {
```

Comment:
Very good idea to return a list of the expected content here 👍 It would be even nicer if we could add a method to our `AbstractMultipleGenerator` which just returns the `ClassName`s that it will generate for a given `TypeElement`. This would then avoid that we have to generate the actual `TypeSpec`s here.